code/modules/paperwork/paperbin.dm

```diff
@@ -120,50 +144,92 @@
+				var/mutable_appearance/paper_overlay = mutable_appearance(current_paper.icon, current_paper.icon_state)
+				paper_overlay.color = current_paper.color
+				paper_overlay.pixel_y = paper_number/8 - 2 //gives the illusion of stacking
+				. += paper_overlay
+				if(paper_number == LAZYLEN(papers)) //this is our top paper
+					. += current_paper.overlays //add overlays only for top paper
+					if(istype(src, /obj/item/paper_bin/bundlenatural))
+						bin_overlay.pixel_y = paper_overlay.pixel_y //keeps binding centred on stack
+					if(bin_pen)
+						pen_overlay.pixel_y = paper_overlay.pixel_y //keeps pen on top of stack
+			paper_number++
+		. += bin_overlay
+
 	if(bin_pen)
-		. += mutable_appearance(bin_pen.icon, bin_pen.icon_state)
+		. += pen_overlay
 
 /obj/item/paper_bin/construction
 	name = "construction paper bin"
 	desc = "Contains all the paper you'll never need, IN COLOR!"
-	icon_state = "paper_binc"
 	papertype = /obj/item/paper/construction
 
 /obj/item/paper_bin/bundlenatural
 	name = "natural paper bundle"
 	desc = "A bundle of paper created using traditional methods."
-	icon_state = "paper_bundle"
+	icon_state = null
 	papertype = /obj/item/paper/natural
 	resistance_flags = FLAMMABLE
+	bin_overlay_string = "paper_bundle_overlay"
+	///Cable this bundle is held together with.
+	var/obj/item/stack/cable_coil/binding_cable
+
+/obj/item/paper_bin/bundlenatural/Initialize(mapload)
+	binding_cable = new /obj/item/stack/cable_coil(src, 2)
+	binding_cable.color = "#a9734f"
+	binding_cable.cable_color = "brown"
+	binding_cable.desc += " Non-natural."
+	return ..()
+
+/obj/item/paper_bin/bundlenatural/dump_contents(atom/droppoint)
+	. = ..()
+	qdel(src)
+
+/obj/item/paper_bin/bundlenatural/update_overlays()
+	bin_overlay = mutable_appearance(icon, bin_overlay_string)
+	bin_overlay.color = binding_cable.color
+	return ..()
 
 /obj/item/paper_bin/bundlenatural/attack_hand(mob/user, list/modifiers)
 	..()
-	if(total_paper < 1)
-		qdel(src)
+	if(!LAZYLEN(papers))
+		deconstruct(FALSE)
+
+/obj/item/paper_bin/bundlenatural/deconstruct(disassembled)
+	dump_contents()
+	qdel(src)
 
 /obj/item/paper_bin/bundlenatural/fire_act(exposed_temperature, exposed_volume)
 	qdel(src)
 
 /obj/item/paper_bin/bundlenatural/attackby(obj/item/W, mob/user)
+	if(istype(W, /obj/item/paper/carbon) && (W.icon_state == "paper_stack" || W.icon_state == "paper_stack_words"))
+		to_chat(user, "<span class='warning'>[W] won't fit into [src].</span>")
+		return
 	if(W.get_sharpness())
-		to_chat(user, "<span class='notice'>You snip \the [src], spilling paper everywhere.</span>")
-		var/turf/T = get_turf(src.loc)
-		while(total_paper > 0)
-			total_paper--
-			var/obj/item/paper/P
-			if(papers.len > 0)
-				P = papers[papers.len]
-				papers -= P
-			else
-				P = new papertype()
-				P.forceMove(T)
-			CHECK_TICK
-		qdel(src)
+		if(W.use_tool(src, user, 1 SECONDS))
+			to_chat(user, "<span class='notice'>You slice the cable from [src].</span>")
```

Comment:
Did you intend to add the cable stuff here? to be clear, I'm fine with it, I'd have merged your last pr, just want to make sure

Reply:
Yeah, I did. I copied the whole proc over from the other PR to replace the while loop, which brought the use_tool over. It just makes it take a second.